.github/workflows/dailyWorkflow.yml

```diff
@@ -47,36 +47,21 @@ jobs:
           uses: ruby/setup-ruby@v1
           with:
             ruby-version: '3.2'
-        - name: Install dependencies
+        - name: Generate report
           run: |
-            python -m pip install --upgrade pip
-            pip install -r requirements.txt
```

Comment:
Why was the install dependencies section removed? it's not clear to me

Reply:
Python is not used in this workflow, so there is no need for us to install python packages